ExtensionSamples/ExtensionSamples/ExtensionSamples.csproj

```diff
@@ -31,7 +31,6 @@
     <Profiling>true</Profiling>
     <PlatformTarget>x86</PlatformTarget>
     <CodeSignProvision>Automatic</CodeSignProvision>
-    <PackageSigningKey>3rd Party Mac Developer Installer</PackageSigningKey>
```

Comment:
It's a default value that VSfM adds after you opened the project signing window. Should we remove it?

Reply:
@mykyta-bondarenko-gl VSfM adds it even for Debug configurations?

Reply:
@rolfbjarne yes, it does. Checked on this PR.

Reply:
@mykyta-bondarenko-gl OK, I've reverted this particular change.

That said, VSfM shouldn't write out default values 😒